regress/netcat.c

```diff
@@ -495,7 +495,7 @@ main(int argc, char *argv[])
 					sv = NULL;
 				else {
 					sv = getservbyport(
-					    ntohs(atoi(portlist[i])),
+					    ntohs((int)strtol(portlist[i], NULL, 10)),
```

Comment:
what warning is this fixing?  atoi already returns an int which is what's needed.